LMGeocoderDemo/LMGeocoder/LMViewController.m

```diff
@@ -119,6 +126,21 @@ - (void)locationManager:(CLLocationManager *)manager didUpdateLocations:(NSArray
                                         }];
 }
 
+- (NSString *)cacheFilePath {
+
+    NSString *documentDir = [NSSearchPathForDirectoriesInDomains(NSDocumentDirectory, NSUserDomainMask, YES) firstObject];
+    return [documentDir stringByAppendingPathComponent:@"abcdef"];
```

Comment:
Is this supposed to be like that? ``abcdef`` ???

Reply:
Yes, its just for testing, 
It's a demo project not related to the lib it self